.github/i18n/README.ru.md

```diff
@@ -148,7 +148,7 @@ from jina.flow import Flow
 f = Flow.load_config('index.yml')
 
 with f:
-    f.index(input_fn)
+    f.index(inputs)
```

Comment:
where do the inputs come from?

Reply:
I think it is just to illustrate the basic structure of a Jina program